test/data/deliusClientTest.js

```diff
@@ -0,0 +1,57 @@
+const nock = require('nock')
+
+const config = require('../../server/config')
+const createDeliusClient = require('../../server/data/deliusClient')
+
+describe('deliusClient', () => {
+  let fakeDelius
+  let deliusClient
+  let signInService
+
+  beforeEach(() => {
+    fakeDelius = nock(`${config.delius.apiUrl}`)
+    signInService = {
+      getAnonymousClientCredentialsTokens: sinon.stub().resolves('token'),
+    }
+    deliusClient = createDeliusClient(signInService)
+  })
+
+  afterEach(() => {
+    nock.cleanAll()
+  })
+
+  describe('deliusClient', () => {
+    it('should throw error on GET when no token', () => {
+      signInService.getAnonymousClientCredentialsTokens.resolves(null)
+      return expect(deliusClient.getROPrisoners('1')).to.be.rejectedWith('Unauthorised access')
+    })
+  })
+
+  describe('getROPrisoners', () => {
+    it('should return data from api', () => {
+      fakeDelius.get(`/staff/staffCode/1/managedOffenders?current=true`).reply(200, { key: 'value' })
+
+      return expect(deliusClient.getROPrisoners('1')).to.eventually.eql({ key: 'value' })
+    })
+
+    it('should reject if api fails', () => {
+      fakeDelius.get(`/staff/staffCode/1/managedOffenders?current=true`).reply(500)
+
```

Comment:
Same thing - ?curren=true added by the proxy. Lets discuss Monday to see where the best place for this is?